README.md

```diff
@@ -93,10 +94,14 @@ The following is a mostly complete list of the software installed in Dad's Garag
 ### Other
 
 - [aws-cli](https://aws.amazon.com/cli/)
+- [azure-cli](https://docs.microsoft.com/en-us/cli/azure/get-started-with-azure-cli)
 - [k9s](https://github.com/derailed/k9s)
 - [kubectl](https://kubernetes.io/docs/reference/kubectl/overview/)
 - [pipx](https://github.com/pipxproject/pipx)
+- [helm 2](https://v2.helm.sh/docs/)
+- [helm 3](https://helm.sh/docs/)
```

Comment:
Can you add helmfile here as well?

Reply:
I have added in the helm docs through it would be more related there.